### PR TITLE
Kpi updates from UAT

### PR DESCRIPTION
**Pull request by jasper-lyons**
The contains updates to a few KPIs that I didn't quite get right from the original spec. Through iteration with the KPI UAT team we've got things to a stable, useful state.

### CODE REVIEW

File: app/javascript/components/key-performance-indicators/component.jsx [hunk 1]
```diff
@@ -180,7 +173,7 @@ const KeyPerformanceIndicators = () => {
 
                 <Grid container spacing={2}>
                   <Grid item className={css.grow} xs={12} md={12}>
-                    <SupervisorToCaseworkerRatio dateRanges={allDateRanges} />
+                    <SupervisorToCaseworkerRatio dateRanges={[]} />
```
Comment (jtoliver-quoin) [inline app/javascript/components/key-performance-indicators/component.jsx:176]: In the SupervisorToCaseworkerRatio component you can default dataRanges  to []. `Component = ({ dataRanges = []}) => {...` That way you won't need to pass a prop here.

File: app/javascript/components/key-performance-indicators/utils/utc-date.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+export default class UTCDate {
```
Comment (jtoliver-quoin) [inline app/javascript/components/key-performance-indicators/utils/utc-date.js:1]: We have the datefns lib in primero... Does it already do some of this?

File: app/javascript/components/key-performance-indicators/component.jsx (hunk 1 above)
Comment (jasper-lyons) [reply app/javascript/components/key-performance-indicators/component.jsx]: Good catch. Done.

File: app/javascript/components/key-performance-indicators/utils/utc-date.js (hunk 2 above)
Comment (jasper-lyons) [reply app/javascript/components/key-performance-indicators/utils/utc-date.js:1]: Yeah, I think I figured out that we don't need any of this anyway 😅. The issue turned out to be when displaying the dates, not when capturing them.

Comment (jasper-lyons) [reply app/javascript/components/key-performance-indicators/utils/utc-date.js:1]: Ok, I see why this is here now. All date math in js (including datefns afaik) is in the clients local timezone so when ever you want the beginning of a month you'll get e.g. `2021/08/01T00:00:00` and then when I convert that to UTC to send to the server it becomes `2021/07/31T23:00:00` which is the wrong month.

I'm reluctant to re-write it now at the 11th hour as this solution has been fairly rigorously put through QA but perhaps we could circle back to it with a datefns based solution? I too would prefer to use it.

You can peek at the commit that introduced this here: ca1e0f9ceb2be34e62eb78d7b59a6f7b0cb909b8 and a previous implementation here: 2a199d06c78ddda684cd13d667303e28fb1ba7e3

File: app/models/kpi/gbv_incident.rb [hunk 3]
```diff
@@ -7,6 +7,7 @@ module Kpi::GBVIncident
   included do
     searchable do
       integer :reporting_delay_days
+      date :date_of_first_report
```
[the remark this answers is not in the document]
Comment (jasper-lyons) [reply app/models/kpi/gbv_incident.rb]: great suggestion and done!

**Review by pnabutovsky — APPROVED** [review]

**Pull request merged**